src/govuk/components/character-count/template.test.js

```diff
@@ -99,9 +99,11 @@ describe('Character count', () => {
     it('renders with hint', () => {
       const $ = render('character-count', {
         id: 'character-count-with-hint',
+        maxlength: 10,
         hint: {
           text: 'It’s on your National Insurance card, benefit letter, payslip or P60. For example, ‘QQ 12 34 56 C’.'
-        }
+        },
+        countMessageClasses: 'app-extra-class'
```

Comment:
I think this should probably be tested in a separate assertion – this test is looking at whether a separate actual 'hint' can be added to the character count ("'It’s on your National Insurance card, benefit letter, payslip or P60. For example, ‘QQ 12 34 56 C’.'") whereas this class is added to the count message, which is unrelated to the rest of the test. I'd also assert that the class is on the element, rather than using a snapshot for it.